packages/db-authorization/test/types/index.test-d.ts

```diff
@@ -0,0 +1,16 @@
+import fastify, {
+  type FastifyPluginAsync
+} from 'fastify'
+import { expectType } from 'tsd'
+import auth, {
+  type DBAuthorizationPluginInterface,
+  type DBAuthorizationPluginOptions
+} from '../..'
+
+expectType<FastifyPluginAsync<DBAuthorizationPluginOptions>>(auth)
+
+const app = fastify()
+app.register(auth)
+app.register(async (instance) => {
+  expectType<DBAuthorizationPluginInterface>(instance.platformatic)
```

Comment:
this test fails because:

```
Parameter type DBAuthorizationPluginInterface is declared too wide for argument type PlatformaticApp.
```
I'm a bit new to types testing so if someone has a good idea on what to do...